blocks/library/video/index.js

```diff
@@ -141,10 +141,9 @@ registerBlockType( 'core/video', {
 	save( { attributes } ) {
 		const { src, caption, align } = attributes;
 		return (
-
 			<figure className={ align ? `align${ align }` : null }>
 				{ src && <video controls src={ src } /> }
-				{ caption && caption.length > 0 && <figcaption>{ caption }</figcaption> }
+				{ caption && caption.length > 0 && <figcaption><Editable.Value value={ caption } /></figcaption> }
```

Comment:
Should we now handle move this logic to Editable.Value?

Reply:
Yes, great idea. It will handle this case with no additional code.

Reply:
Actually, this is for the `<figcaption>`, didn't see that one. So we probably can't remove it.

Reply:
Unless we create another component like:
```js
<Editable.Tag name="figcaption" value={ caption } />
```

It would fly nicely also with the following code:
```js
const Tag = nodeName.toLowerCase();

return <Tag><Editable.Value value={ values } /></Tag>;
```

which would get simplified to:
```js
return <Editable.Tag name={ nodeName } value={ values } />;
```
assuming we call `toLowerCase` inside the component.

Reply:
I personally don't see it being an issue to be explicit about this. Maybe simplified with Lodash `isEmpty`